alarm_control_panel-card/alarm_control_panel-card.js

```diff
@@ -323,17 +323,13 @@ class AlarmControlPanelCard extends HTMLElement {
     return style;
   }
 
-  _stateToIcon(state) {
-    return this._icons[state] || 'mdi:shield-outline'
-  }
-
   _stateToText(state) {
-    return this._stateStrings[state] || 'Unknown'
+    return state.split('_').join(' ').replace(/^\w/, c => c.toUpperCase());
```

Comment:
What about leaving the change in for now since it is fully functional, and...

The next refactoring, when an approach is figured out, is to include the translation file. This could still be a fallback, or not. Knowing the translation approach may open additional possibilities.

In addition to the translation file being loaded it would be good to have configuration allowing mapping of states (etc.) to display strings. That would handle the case when there are custom states and custom actions. 

The concept I was thinking about was something such as this:
```
labels:
  alarm_code: Enter 4 digits alarm code
  state_home_instant: Home
```

Reply:
I agree we could allow labels so people can translate it themselves, awesome idea! 

Maybe even just define static some default labels (like we used to) and allow people to define what is custom for their component. I think we should keep component at a minimum and allow extensions like this.